packages/form-core/src/FormApi.ts

```diff
@@ -1956,6 +1959,36 @@ export class FormApi<
     this.validateField(`${field}[${index1}]` as DeepKeys<TFormData>, 'change')
     this.validateField(`${field}[${index2}]` as DeepKeys<TFormData>, 'change')
   }
+
+  /**
+   * Resets the field value and meta to default state
+   */
+  resetField = <TField extends DeepKeys<TFormData>>(field: TField) => {
+    this.baseStore.setState((prev) => {
+      return {
+        ...prev,
+        fieldMetaBase: {
+          ...prev.fieldMetaBase,
+          [field]: {
+            isValidating: false,
+            isTouched: false,
+            isBlurred: false,
+            isDirty: false,
+            isPristine: true,
+            errors: [],
+            errorMap: {},
+          },
```

Comment:
We have these values elsewhere in the `getDefaultMeta` or something, don't we? Let's reuse to avoid the bundle size increase